packages/gatsby-plugin-google-tagmanager/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "gatsby-plugin-google-tagmanager",
   "description": "Gatsby plugin to add google tagmanager onto a site",
-  "version": "2.0.7",
+  "version": "2.0.8",
```

Comment:
Please revert that. Package version is being bumped automatically during our publishing process